app/Filament/Resources/AchievementResource.php

```diff
@@ -244,8 +244,8 @@ public static function table(Table $table): Table
                 Tables\Columns\TextColumn::make('Flags')
                     ->badge()
                     ->formatStateUsing(fn (int $state): string => match ($state) {
-                        AchievementFlag::OfficialCore => __('published'),
-                        AchievementFlag::Unofficial => __('unpublished'),
+                        AchievementFlag::OfficialCore => 'Published',
+                        AchievementFlag::Unofficial => 'Unpublished',
```

Comment:
The audit entries for this field say "3->5". Can we clean format those?

Reply:
This was far less trivial than I hoped it would be, but it should be working now in latest:

![Screenshot 2024-07-17 at 7 46 03 PM](https://github.com/user-attachments/assets/f19aac5f-6d9e-4452-b077-d2fad3ba9212)